apps/evans/components/core/AudioPlayer.tsx

```diff
@@ -20,6 +20,27 @@ const AudioPlayer: React.FC<Props> = ({ tracks }) => {
   const [audio, setAudio] = useState<HTMLAudioElement | null>(null);
   const track = tracks[currentTrackIndex];
 
+  useEffect(() => {
+    const trackStorageItem = localStorage.getItem(window.location.pathname);
```

Comment:
let's change the key just a bit to reflect _what_ it's storing, maybe we concatenate the path with some prefix, like `audio_position--` so we get `audio_position--hugh-turford/walk-in-the-spirit` as our key